src/mono/wasi/include/pthread.h

```diff
@@ -0,0 +1,226 @@
+// The WASI SDK doesn't include pthread.h, but the definitions are required by various parts of the Mono sources
```

Comment:
What is the license for this file ?

Reply:
I've added further comments to the file headers to clarify that it's dual-licensed (Apache-2.0 and MIT).